tests/nightly/test_large_array.py

```diff
@@ -28,7 +28,6 @@
 SMALL_Y = 50
 LARGE_SIZE = LARGE_X * SMALL_Y
 
-
```

Comment:
keep it. It's Pep8 style

Reply:
Sure! Are you referring to this: https://www.python.org/dev/peps/pep-0008/#id21
Please verify.

Reply:
yes